src/backend/gporca/libgpopt/src/xforms/CXformFactory.cpp

```diff
@@ -405,18 +392,18 @@ CXformFactory::Init()
 void
 CXformFactory::Shutdown()
 {
-	CXformFactory *pxff = CXformFactory::Pxff();
-
-	GPOS_ASSERT(NULL != pxff && "Xform factory has not been initialized");
-
-	CMemoryPool *mp = pxff->m_mp;
+	if (NULL != m_pxff)
```

Comment:
I suggest to use
```
if (NULL == m_pxff)
    return;
```
because this way we can reduce indentation and number of modified lines

Reply:
I did it in a similar way to https://github.com/arenadata/gpdb/blob/db2d3880fb989ef49afa121d13869f768fd2c901/src/backend/gporca/libgpos/src/common/CDebugCounter.cpp#L72-L83

Reply:
Why is it better than the variant which I suggest?

Reply:
> Why is it better than the variant which I suggest?

It's not better, it's just that it's already been done this way in the specified place, and I did it by analogy as well.

Reply:
done